# Type: test
# Subtype: T1
# English Summary: Remove unnecessary code, rename test methods
# Chinese Summary: 删除无用代码，重命名测试方法
# Entry Number: 15

# === Git Diff Content ===

diff --git a/community/server/src/functionaltest/java/org/neo4j/server/rest/ManageNodeFunctionalTest.java b/community/server/src/functionaltest/java/org/neo4j/server/rest/ManageNodeFunctionalTest.java
index 2395464..585f945 100644
--- a/community/server/src/functionaltest/java/org/neo4j/server/rest/ManageNodeFunctionalTest.java
+++ b/community/server/src/functionaltest/java/org/neo4j/server/rest/ManageNodeFunctionalTest.java
@@ -170,11 +170,6 @@ public class ManageNodeFunctionalTest
 
     }
 
-    private void assertProperNodeRepresentation( final Map<String, Object> noderep )
-    {
-        NodeRepresentationTest.verifySerialisation( noderep );
-    }
-
     @Test
     public void shouldRespondWith204WhenNodeDeleted() throws Exception
     {
@@ -183,7 +178,7 @@ public class ManageNodeFunctionalTest
     }
 
     @Test
-    public void shouldRespondWith404WhenNodeToBeDeletedCannotBeFound() throws Exception
+    public void shouldRespondWith404AndSensibleEntityBodyWhenNodeToBeDeletedCannotBeFound() throws Exception
     {
         ClientResponse response = sendDeleteRequestToServer( NON_EXISTENT_NODE_ID );
         assertEquals( 404, response.getStatus() );
